transactions/idTableStaking/register_node.cdc

```diff
@@ -28,6 +28,8 @@ transaction(id: String,
         
         if acct.borrow<&FlowIDTableStaking.NodeStaker>(from: FlowIDTableStaking.NodeStakerStoragePath) == nil {
             acct.save(<-nodeStaker, to: FlowIDTableStaking.NodeStakerStoragePath)
+
+            acct.link<&{FlowIDTableStaking.NodeStakerPublic}>(FlowIDTableStaking.NodeStakerPublicPath, target: FlowIDTableStaking.NodeStakerStoragePath)
```

Comment:
I noticed that `NodeStakerPublicPath` was already defined. Was it being used for something else before? Should there be a similar constant for `NodeDelegator` (`NodeDelegatorPublicPath`)?

Reply:
It was only being used for testing before, but now it is useful in real life! We can't add a `NodeDelegatorPublicPath` though because we can't add any state variables in an upgrade. We could add a getter function for it though that just returns the path we want. What do you think?

Reply:
Ah I see, didn't realize that was a restriction! I think the way you have it is fine